alacritty_terminal/Cargo.toml

```diff
@@ -0,0 +1,95 @@
+[package]
+name = "alacritty_terminal"
+version = "0.3.2"
+authors = ["Joe Wilm <joe@jwilm.com>"]
+license = "Apache-2.0"
+build = "build.rs"
+description = "GPU-accelerated terminal emulator - library for embedders"
```

Comment:
I think I've mentioned this previously but you've probably missed it when including my changes, but this library is completely unrelated to GPU acceleration.

In my PR I've used the following description:
```
description = "Library for writing terminal emulators"
```

I have added a bunch of little things to the `Cargo.toml` files like keywords and categories that should help with separating the different subcrates, even though it's not necessary I think these might be helpful to just add here.

Reply:
Done